project/src/main/webapp/index.html

```diff
@@ -3,41 +3,62 @@
   </head>
-  <body onload="">
+  <body class="text-content">
     <h1> FoodMood - your food order helper!</h1>
-    <form action="/results.html">
-      <label for="cuisine">Choose a cuisine:</label>
-      <select class="button" name="cuisine" id="cuisine">
-        <option value="all">all cuisines</option>
-        <option value="Italian">Italian</option>
-        <option value="Asian">Asian</option>
-        <option value="Sushi">Sushi</option>
-        <option value="Hamburger">Hamburger</option>
-        <option value="Salad">Salad</option>
-        <option value="Homemade">Homemade</option>
-      </select>
-      <br><br>
-      <label for="rating">Min rating:</label>
+    <div id="query-form" class="query-form">
+      <form>
+        <p>Select cuisines:</p>
+        <input type="checkbox" name="all-cuisines" value="all">all cuisines
+        <input type="checkbox" name="cuisine" value="Italian">Italian
+        <input type="checkbox" name="cuisine" value="Asian">Asian
+        <input type="checkbox" name="cuisine" value="Sushi">Sushi
+        <input type="checkbox" name="cuisine" value="Hamburger">Hamburger
+        <input type="checkbox" name="cuisine" value="Salad">Salad
+        <input type="checkbox" name="cuisine" value="Homemade">Homemade
+      </form>
+      <br>
+      <label for="rating" class="text-content">Min rating:</label>
       <select class="button" name="rating" id="rating">
+        <option value="0" selected="selected">I don't mind</option>
         <option value="5">5 stars out of 5</option>
-        <option value="4" selected="selected">4 stars out of 5</option>
+        <option value="4">4 stars out of 5</option>
         <option value="3">3 stars out of 5</option>
         <option value="2">2 stars out of 5</option>
         <option value="1">1 stars out of 5</option>
       </select>
       <br><br>
-      <label for="price">price range:</label>
+      <label for="price" class="text-content">price range:</label>
       <select class="button" name="price" id="price">
-        <option value="low">low budget</option>
-        <option value="normal" selected="selected">normal</option>
-        <option value="high">wiiling to spend a little extra</option>
-        <option value="very high">no budget limit</option>
+        <option value="5" selected="selected">I don't mind</option>
+        <option value="0">low budget</option>
+        <option value="1">normal</option>
+        <option value="2">wiiling to spend a little extra</option>
+        <option value="3">no budget limit</option>
       </select>
       <br><br>
-      <input class="submit" type="submit" value="Decide for me!">
-    </form>
+      <label for="location" class="text-content">Location:</label>
+      <input class="button" type="text" id="location" name="location">
+      <br><br>
+      <input class="submit" type="submit" value="Decide for me!" onclick="fetchFromQuery()">
+    </div>
+    <div id="results" style="display: none">
```

Comment:
Good solution to the redirecting issue to put it all in one page. So if I understand correctly, regarding my task to make the results page all is left is to add the map?

Reply:
yes exactly! :)